projects/owi/programs/owi/tests/basic.nix

```diff
@@ -0,0 +1,28 @@
+{
+  sources,
+  ...
+}:
+
+{
+  name = "Program Name";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.owi
+          sources.examples.owi.basic
+        ];
+      };
+  };
+
+  testScript =
+    { nodes, ... }:
+    ''
+      start_all()
+
+      machine.succeed("owi version")
```

Comment:
Hm this will be a rather expensive test for something that can run in the derivation. Let's better keep that out.

Reply:
Sure thing. That's what I thought too. I included it because the majority (if not all) of the other projects in ngipkgs do this too. We might want to create an issue documenting and figuring out how to replace it. 

On that note, what should I replace it with? The shell already has some comprehensive tests.

Reply:
@themadbit and @eljamm were discussing how to reuse samples across demos IIUC, so maybe we can align that somehow? And if not, let's just skip it for now (set to null) and figure it out later.